sdk/inc/azure/iot/az_iot_hub_client_properties.h

```diff
@@ -55,7 +55,7 @@
  * @return An #az_result value indicating the result of the operation.
  * @retval #AZ_OK The JSON payload was prefixed successfully.
  */
-AZ_NODISCARD az_result az_iot_hub_client_properties_builder_begin_component(
+AZ_NODISCARD az_result az_iot_hub_client_properties_writer_begin_component(
```

Comment:
I am assuming this is the first "beta" for such APIs. Otherwise, we may want to have some sort of CL entry to let users know what is being changed/added in this release.

Similar to:
https://github.com/Azure/azure-sdk-for-c/releases/tag/1.1.0-beta.2

Reply:
Yes.  I'll create some text around this and provide it to you all for release/packaging.